apps/root/src/pages/transfer/components/transfer-form/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import styled from 'styled-components';
+import { BackgroundPaper, Divider, Typography, colors } from 'ui-library';
+import NetworkSelector from '@common/components/network-selector';
+import TokenSelector from '../token-selector';
+import RecipientAddress from '../recipient-address';
+import TransferButton from '../transfer-button';
+import useActiveWallet from '@hooks/useActiveWallet';
+import { FormattedMessage } from 'react-intl';
+import { useParams } from 'react-router-dom';
+import useToken from '@hooks/useToken';
+import { isEqual, orderBy } from 'lodash';
+import { useAppDispatch } from '@hooks/state';
+import { useTransferState } from '@state/transfer/hooks';
+import { setChainId, setRecipient, setToken } from '@state/transfer/actions';
+import { identifyNetwork, validateAddress } from '@common/utils/parsing';
+import { getAllChains } from '@mean-finance/sdk';
+import { NETWORKS } from '@constants';
+import useReplaceHistory from '@hooks/useReplaceHistory';
+import { useThemeMode } from '@state/config/hooks';
+
+const StyledTransferForm = styled(BackgroundPaper)`
```

Comment:
does not the background paper already this styles? You can use elevation=0 for it to be outlined

Reply:
I was able to remove some styles, but couldn't find a way to get the border with a native paper prop

Reply:
what about the padding? is it different from our paper definition?

Reply:
What about the variant outline of the paper? That should get you the border https://mui.com/material-ui/react-paper/#variants

Reply:
I have added this to our theme succesfully 😄
Taking a deeper look at our design definitions, I have implemented a modified `BackgroundPaper` at our activity, net worth and transfer. Most of them were elevated, and they should be outlined

Reply:
nice!